frontend/sw360-portlet/src/main/webapp/html/components/editRelease.jsp

```diff
@@ -43,6 +43,7 @@
 <liferay-theme:defineObjects/>
 
 <c:catch var="attributeNotFoundException">
+    <jsp:useBean id="isUserAtLeastClearingAdmin" class="java.lang.String" scope="request" />
```

Comment:
It should be possible to have Boolean-beans like on `/sw360-portlet/src/main/webapp/html/components/detail.jsp` the bean `<jsp:useBean id="isUserAllowedToMerge" type="java.lang.Boolean" scope="request"/>`.

Reply:
@maxhbr and @imaykay 
Changed `isUserAtLeastClearingAdmin` to Boolean bean.
Added back-end validation for Mainline State change through web UI and REST interface as well.